src/components/cylc/Info.vue

```diff
@@ -0,0 +1,261 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <div class="c-info">
+    <!-- The task summary
+
+    * The "-40" comes from the GraphNode offset, see the GraphNode compoent.
+    * The height is "200" + "40".
+    * The "99999" is the maximum component width (because we don't know
+      how wide it will be until we render it).
+    * The "overflow-x: hidden" prevents this 99999 width from causing
+      horizontal scrolling.
+    * The "8em" is the height that this SVG will be scaled to fill.
+    -->
+    <div style="overflow-x: hidden;">
+      <svg
+        preserveAspectRatio="xMinYMin"
+        viewBox="-40 -40 99999 200"
+        height="6em"
+      >
+        <GraphNode
+          :task="task"
+          :jobs="task.children"
+          :jobTheme="jobTheme"
+        />
```

Comment:
Something is going wrong with the command menu. It does not know what type of Cylc object it is representing (task or cycle or workflow etc.)

![image](https://github.com/user-attachments/assets/aa33543d-84ed-44f2-b74b-4fa467bc78e8)

Reply:
Curious, will take a look.

I'm guessing the graph view doesn't show this issue?

Reply:
Sorted.

The data-store node standin thinggy was missing the `type` field which the menu apparently uses.